src/main/java/org/pih/petl/job/RunMultipleJob.java

```diff
@@ -21,7 +21,7 @@
     @Override
     public void execute(final ExecutionContext context) throws Exception {
         JobConfigReader configReader = new JobConfigReader(context);
-        JobExecutor jobExecutor = new JobExecutor(configReader.getInt(1, "maxConcurrentJobs"));
+        JobExecutor jobExecutor = new JobExecutor(1);
```

Comment:
So we are removing the "maxConcurrentJobs" parameter here?  (No strong objections, just trying to understand)

Reply:
Yeah, just removing it from the pipeline job and keeping it simply - it's just a way to run a series of jobs one after another.